projects/hal-components/src/lib/components/popup-connected/popup-connected.component.ts

```diff
@@ -13,45 +11,66 @@ export type popupPosition = 'above' | 'below';
 })
 export class PopupConnectedComponent implements OnInit, OnChanges {
 
-  position: ConnectionPositionPair = new ConnectionPositionPair(
-    { originX: 'end', originY: 'bottom' },
-    { overlayX: 'end', overlayY: 'top' }
-  );
-
   @Input() small = false;
   @Input() isOpen = false;
   @Input() relativePositionY: popupPosition = 'below';
   @Input() origin: CdkOverlayOrigin;
+  @Input() alignedRight = false;
```

Comment:
Tror vi bør nytte til alignRight 

Og enten må vi endre relativePositionY til allignTop, eller endre alignRight til relativePositionX. Stemmer for allignTop og alignRight. Det blir en liten breaking, men da får vi samme interface for begge

Reply:
Enig, har endret relativePositionY til alignTop nå. Endret også alignTop til å ta imot en boolean istedenfor så begge 'posisjons' inputene er er like i navn og inputtype